OCR Project 4

crm-ops opened the pull request:
Following comments from @fpatelm we are proposing the following changes :

- will delete crm-ops/sfdc-ocr-p4 cloned project
- we have forked  Project_DA_Java_EN_Come_to_the_Rescue_of_a_Java_Application
- AnalyticsCounter is now only a main method
- ISymptomReader interface is now implemented in ReadSymptomDataFromFile
- New class SymptomsCounter - does the map/reduce count operation
- New class SymptomsFileOutput - write the result ordered alphabetically
- Map object are used when necessary for code simplicity

Commit:
OCR Project 4
Following comments from @fpatelm we are proposing the following changes :

- will delete crm-ops/sfdc-ocr-p4 cloned project
- we have forked  Project_DA_Java_EN_Come_to_the_Rescue_of_a_Java_Application
- AnalyticsCounter is now only a main method
- ISymptomReader interface is now implemented in ReadSymptomDataFromFile
- New class SymptomsCounter - does the map/reduce count operation
- New class SymptomsFileOutput - write the result ordered alphabetically
- Map object are used when necessary for code simplicity

fpatelm commented on Project02Eclipse/src/com/hemebiotech/analytics/AnalyticsCounter.java:
```diff
@@ -1,43 +1,63 @@
+		SymptomsCounter symptomcounter = new SymptomsCounter(inputSymptoms);
+		symptomsreport = symptomcounter.getSymptomsCount();
+
+
```
remove empty lines

crm-ops replied:
Done on second commit

fpatelm commented on Project02Eclipse/src/com/hemebiotech/analytics/SymptomsCounter.java:
```diff
@@ -0,0 +1,62 @@
+
+        }
+
+
```
remove empty lines

crm-ops replied:
done on second commit

fpatelm commented on Project02Eclipse/src/com/hemebiotech/analytics/SymptomsCounter.java:
```diff
@@ -0,0 +1,62 @@
+
+                results.replace(str,results.get(str)+1);
+            }
+
```
remove empty lines

crm-ops replied:
done on second commit

fpatelm reviewed (changes requested):
Just removing empty lines as per review. Good Job!

Commit:
Removed empty lines

OCR Project 4
Following comments from @fpatelm we are proposing the following changes :

- will delete crm-ops/sfdc-ocr-p4 cloned project
- we have forked  Project_DA_Java_EN_Come_to_the_Rescue_of_a_Java_Application
- AnalyticsCounter is now only a main method
- ISymptomReader interface is now implemented in ReadSymptomDataFromFile
- New class SymptomsCounter - does the map/reduce count operation
- New class SymptomsFileOutput - write the result ordered alphabetically
- Map object are used when necessary for code simplicity

fpatelm commented:
Approved
